tests/context/conftest.py

```diff
@@ -242,3 +242,900 @@ def ce_details_dynamic_event_deleted(ce_details_dynamic_event):
+            "firstName": "john",
+            "id": "258b95c5-1e48-4357-881c-ad4d544eab32",
+            "lastName": "doe",
+            "sampleUnitType": "BI",
+            "status": "ACTIVE",
+            "telephone": "07772257772",
+            "enrolmentStatus": "ENABLED",
+        },
+        {
+            "associations": [
+                {
+                    "businessRespondentStatus": "ACTIVE",
+                    "enrolments": [{"enrolmentStatus": "ENABLED", "surveyId": "02b9c366-7397-42f7-942a-76dc5876d86d"}],
+                    "partyId": "3d6597e3-2bee-43d7-84e0-6f4f993240eb",
+                    "sampleUnitRef": "49900000001",
+                }
+            ],
+            "emailAddress": "test@example.com",
+            "firstName": "test",
+            "id": "985bf97e-4f03-4898-92ef-dd7aac23ab08",
+            "lastName": "test",
+            "sampleUnitType": "BI",
+            "status": "ACTIVE",
+            "telephone": "1234567890",
+            "enrolmentStatus": "ENABLED",
+        },
+    ]
+
+
+@pytest.fixture
+def suspended_survey_respondents(survey_respondents):
+    suspended_survey_respondents = survey_respondents.copy()
+    suspended_survey_respondents[0]["status"] = "SUSPENDED"
+    return suspended_survey_respondents
+
+
+@pytest.fixture
+def pending_enrolment_survey_respondents(survey_respondents):
+    pending_enrolment_survey_respondents = survey_respondents.copy()
+    pending_enrolment_survey_respondents[0]["enrolmentStatus"] = "PENDING"
+    return pending_enrolment_survey_respondents
+
+
+@pytest.fixture
+def disabled_enrolment_survey_respondents(survey_respondents):
+    disabled_enrolment_survey_respondents = survey_respondents.copy()
+    disabled_enrolment_survey_respondents[0]["enrolmentStatus"] = "DISABLED"
+    return disabled_enrolment_survey_respondents
+
+
+@pytest.fixture()
+def case():
+    return {
+        "state": "ACTIONABLE",
+        "id": "f4056be6-2581-4308-b7cd-88118325e81d",
+        "actionPlanId": None,
+        "activeEnrolment": True,
+        "collectionInstrumentId": "e726f50c-504c-4958-be96-9d77520746b9",
+        "partyId": "a5348157-feb4-4bad-9614-fc76e2bfea94",
+        "sampleUnitId": "d8579750-8817-4a58-8fa5-867f1447e1cc",
+        "iac": "t7jk7l29hymx",
+        "caseRef": "1000000000000003",
+        "createdBy": "SYSTEM",
+        "sampleUnitType": "B",
+        "createdDateTime": "2024-02-08T11:39:58.199Z",
+        "caseGroup": {
+            "collectionExerciseId": "1012f36c-b352-431c-b0c9-e7f435cbdd0c",
+            "id": "39c7e488-3f06-4ea9-82fb-3c619f362cb6",
+            "partyId": "a5348157-feb4-4bad-9614-fc76e2bfea94",
+            "sampleUnitRef": "49900000001",
+            "sampleUnitType": "B",
+            "caseGroupStatus": "NOTSTARTED",
+            "surveyId": "02b9c366-7397-42f7-942a-76dc5876d86d",
+        },
+        "caseEvents": None,
+    }
+
+
+@pytest.fixture
+def multiple_cases():
+    return {
+        "state": "ACTIONABLE",
```

Comment:
Again this looks like the same chunk bar an odd change

Reply:
Updated this